### ISSUE TITLE
Fixtures from MySQL using SET/ENUM

### ISSUE THREAD

**Issue by dogmatic69** [cakephp/cakephp#2637]
Cake does not support these at the moment which is fine, 

fixtures are baked as 'text' which is also fine.

these are given a default value which breaks the CREATE statement.

Cake swallows the errors (well not directly shown in any case) which makes it difficult to figure out what the problem actually is.

Would it be possible to check the field type is set/enum and remove the default from baked fixtures?

**Comment by lorenzo** [comment]
Closing all enhancement requests that were not actioned in a long time. If you still think this a good idea and would like to help making this happen, please leave a comment.

**Comment by matheuscmpm** [comment]
I know that this is very old - and it was an old problem for me as well... I did some changes in Cakephp 2 files and was able to handle ENUM type fields while working with schema create/generate. 

I don't know if anyone is still needing this, so I posted the solution in this git public repository.

https://github.com/matheuscmpm/cakephp-schema-enum

Cheers. 

PS: I didn't tried or either fixed the SET type field (I don't use them - but I think that the same solution could be applied, it is really simple to replicate btw).
